flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/IcebergTableSink.java

```diff
@@ -63,6 +64,16 @@ public SinkRuntimeProvider getSinkRuntimeProvider(Context context) {
         .map(UniqueConstraint::getColumns)
         .orElseGet(ImmutableList::of);
 
+    if (readableConfig.getOptional(FlinkConfigOptions.TABLE_EXEC_ICEBERG_USE_FLIP143_SINK).orElse(false)) {
```

Comment:
there is no need for `getOptional` and `orElse`. Flink configuration already handles default value.